### PR TITLE
Feature/devteam

### PR DESCRIPTION
**Pull request by Cardamced**
## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [X] Frontend
- [ ] Backend
- [ ] Other: .....

## General checklist:

- [X] My code follows the style guidelines of this project
- [X] I ran `bun run format`/`bun run lint` before commit
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation where needed
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] I have updated `develop` and merged to my branch before submitting pull request
- [X] My pull request generate no conflicts with `develop` branch
- [X] I requested code review from other team members

## Frontend checklist:

- [X] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [X] My `Javascript` generate no new console errors
- [X] I tested my code cross browsers
- [X] My slice is pixel perfect for both desktop and mobile according to design
- [ ] I conducted basic QA to assure all features are working
- [X] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by No3ml3 — DISMISSED** [review]

**Review by jaalazard — DISMISSED** [review]

File: packages/frontend/web/src/components/ShowTeam.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      img2: './justinepic.jpg',
+    },
+    {
+      name: 'Cédrick',
+      linkedIn: 'https://www.linkedin.com/in/c%C3%A9drick-marie/',
+      github: 'https://github.com/Cardamced',
+      img1: './avatar/avatar-9.webp',
+      img2: './cedpic.jpeg',
+    },
+  ];
```
Comment (YDrogen) [inline packages/frontend/web/src/components/ShowTeam.tsx]: à sortir hors du component 🤓

**Review by YDrogen — CHANGES_REQUESTED** [review]

File: packages/frontend/web/src/components/ShowTeam.tsx (hunk 1 above)
Comment (Cardamced) [reply packages/frontend/web/src/components/ShowTeam.tsx]: C'est fait.

**Review by No3ml3 — DISMISSED** [review]

File: packages/frontend/web/public/jaapic.jpg
Comment (jaalazard) [inline packages/frontend/web/public/jaapic.jpg:1]: C'est un non

File: packages/frontend/web/public/jeehapic.jpeg
Comment (jaalazard) [inline packages/frontend/web/public/jeehapic.jpeg:1]: C'est non.

**Review by jaalazard — APPROVED** [review]

**Review by No3ml3 — APPROVED** [review]

**Review by jaalazard — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**